Resources/config/doctrine_orm.xml

```diff
@@ -47,7 +47,7 @@
         </service>
 
         <!-- DatagridBuilder guesser -->
-        <service id="sonata.admin.builder.orm_datagrid" class="Sonata\DoctrineORMAdminBundle\Builder\DatagridBuilder">
+        <service id="sonata.admin.builder.orm_datagrid" class="Sonata\DoctrineORMAdminBundle\Builder\DatagridBuilder" scope="prototype">
```

Comment:
This line is too long (there is an horizontal scrollbar on Github). Please convert to

```xml
<service 
    id="sonata.admin.builder.orm_datagrid" 
    class="Sonata\DoctrineORMAdminBundle\Builder\DatagridBuilder" 
    scope="prototype">
```